Apps/W1/HybridGP/app/src/Migration/History/GPPOPReceiptApply.table.al

```diff
@@ -294,6 +295,27 @@ table 4060 "GPPOPReceiptApply"
         exit(TotalShipped);
     end;
 
+    procedure GetSumQtyShippedByUnitCost(PO_Number: Text[18]; PO_LineNo: Integer; Location: Text[12]; UnitCost: Decimal): Decimal
+    var
+        TotalShipped: Decimal;
+    begin
+        TotalShipped := 0;
+        Rec.Reset();
```

Comment:
It is dangerous to use Rec, because you will delete all filters. It is better to use a different variable and to copy filters or use a rec to filter on the values.